packages/office-addin-dev-settings/src/cli.ts

```diff
@@ -56,13 +56,17 @@ commander
     .description("Configure the runtime log for all Office Add-ins.")
     .action(commands.runtimeLogging);
 
-commander
-    .command("sideload <manifest-path>")
+    commander
+    .command("sideload <manifest-path> [app-type]")
     .description("Launch Office with the Office Add-in loaded.")
-    .option("-a,--app <app>", `The Office app to launch. ("Excel", "Outlook", "PowerPoint", or "Word")`)    
+    .option("-a,--app <app>", `The Office app to launch. ("Excel", "Outlook", "PowerPoint", or "Word")`)
     .option("-d,--document <document>", `The location of the document to be sideloaded - this can be an absolute file path or url`)
-    .option("-t, --type <type>", `Application type to sideload ("desktop" or "web")`)
-    .action(commands.sideload);
+    .action(commands.sideload)
+    .on("--help", () => {
```

Comment:
I would revise the help text. It is not really a "desktop add-in" or a "web add-in". It is whether to load in the Office desktop app or in Office Online (web).

My suggestion:
```
[app-type] specifies the type of Office app:
   'desktop': Office app for Windows or Mac (default),
   'web': Office running in the web browser
 ```

Reply:
updated locally